backwardcompatibilityml/tensorflow/models.py

```diff
@@ -5,6 +5,40 @@
 
 
 class BCNewErrorCompatibilityModel(tf.keras.models.Sequential):
+    """
+    BackwardCompatibility base model for Tensorflow
+
+    One may create a new Tensorflow model by subclassing
```

Comment:
Change instances of "One" to "You." ( per Microsoft style)   "You may create ...

Reply:
Done.